src/js/views/patients/schedule/filters_views.js

```diff
@@ -0,0 +1,37 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import 'sass/modules/buttons.scss';
+
+import Droplist from 'js/components/droplist';
+import OwnerDroplist from 'js/views/patients/shared/components/owner_component';
+
+const FiltersView = View.extend({
+  className: 'worklist-list__filters',
```

Comment:
shouldn't reuse the workist-list classes here.  Either we need to module these or duplicate them.